--- lxml-2.2.6/src/lxml/html/_html5builder.py.wiget	2009-09-11 10:55:58.000000000 +0200
+++ lxml-2.2.6/src/lxml/html/_html5builder.py	2010-03-27 16:04:28.693916731 +0100
@@ -7,6 +7,11 @@ html5lib style guide.
 from html5lib.treebuilders import _base, etree as etree_builders
 from lxml import html, etree
 
+try:
+    empty_string = unicode()
+except NameError:
+    empty_string = str()
+
 
 class DocumentType(object):
 
@@ -77,7 +82,7 @@ class TreeBuilder(_base.TreeBuilder):
                                                   self.doctype.systemId))
             buf.append('>')
         buf.append('<html></html>')
-        root = html.fromstring(u''.join(buf))
+        root = html.fromstring(empty_string.join(buf))
 
         # Append the initial comments:
         for comment in self.initialComments:
--- lxml-2.2.6/src/lxml/html/_diffcommand.py.wiget	2009-09-11 10:55:58.000000000 +0200
+++ lxml-2.2.6/src/lxml/html/_diffcommand.py	2010-03-27 16:02:10.727258598 +0100
@@ -34,7 +34,7 @@ def main(args=None):
     if options.annotation:
         return annotate(options, args)
     if len(args) != 2:
-        print 'Error: you must give two files'
+        print('Error: you must give two files')
         parser.print_help()
         sys.exit(1)
     file1, file2 = args
@@ -82,6 +82,6 @@ def split_body(html):
     return pre, html, post
 
 def annotate(options, args):
-    print "Not yet implemented"
+    print("Not yet implemented")
     sys.exit(1)
     
